server/models/ProfileModel.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
```

Comment:
You can add a reference to the `User` model here :)

Reply:
Ok

Reply:
You can do so like this: `userId: [{ type: Schema.Types.ObjectId, ref: 'user' }]`